exercises/x1.py

```diff
@@ -0,0 +1,45 @@
+import requests
+
+def printResponse(funcName, response):
```

Comment:
According to Python naming conventions this should be `print_response()`. The PEP8 styleguide can be found here: https://www.python.org/dev/peps/pep-0008/

We run `flake8` with some plugins to enforce consistent capitlization and some other things:
- flake8
- flake8-blind-except
- flake8-builtins
- flake8-eradicate
- flake8-import-order